codalab/lib/profiling_util.py

```diff
@@ -0,0 +1,18 @@
+import cProfile
+import pstats
+
+
+def cprofile(sort_args=['cumulative'], print_args=[10]):
```

Comment:
Add docstring describing what this function does / how to use it and any impact on performance.